terraform/infra/common/oidc_gke.tf

```diff
@@ -0,0 +1,17 @@
+# From:
+# https://github.com/mozilla/global-platform-admin/blob/d3ba91536f92d2de15c9fc0f22a216f0d6cadc59/platform-shared/k8s/bootstrap/webservices-high-public-nonprod-us-west1.yaml
+module "oidc_gke_webservices_high_public_nonprod" {
+  source           = "github.com/mozilla/terraform-modules//aws_gke_oidc_config?ref=aws_gke_oidc_config-0.1.0"
+  gcp_region       = "us-west1"
+  gcp_project_id   = "moz-fx-webservices-high-nonpro"
+  gke_cluster_name = "webservices-high-public-nonprod-us-west1"
+}
+
+# From:
+# https://github.com/mozilla/global-platform-admin/blob/d3ba91536f92d2de15c9fc0f22a216f0d6cadc59/platform-shared/k8s/bootstrap/webservices-high-public-prod-us-west1.yaml
+module "oidc_gke_webservices_high_public_prod" {
+  source           = "github.com/mozilla/terraform-modules//aws_gke_oidc_config?ref=aws_gke_oidc_config-0.1.0"
+  gcp_region       = "us-west1"
+  gcp_project_id   = "moz-fx-webservices-high-prod"
+  gke_cluster_name = "webservices-high-public-prod-us-west1"
```

Comment:
This, as well as line 7, will require https://github.com/mozilla/global-platform-admin/pull/3879/commits/1f797a5d705d9dbb193b1759739f0806f35feefe